src/docfx/build/dependency/DependencyResolver.cs

```diff
@@ -123,7 +123,8 @@ public string GetRelativeUrl(Document fileRelativeTo, Document file)
                 return (uidError, uidHref, null, referencedFile);
             }
 
-            var (error, file, redirectTo, query, fragment, isSelfBookmark, _) = TryResolveFile(relativeTo, href);
+            var decodedHref = HttpUtility.UrlDecode(href);
```

Comment:
Why not HtmlDecode here? In my understanding, it needs urldecode+htmldecode to get file from href.

Reply:
html decode/encode is handled by markdig, our custom `HtmlUtility.TransformLinks` handles html encode/decode itself.

You can verify the behavior from tests, `a&b` and `a&amp;b` all links to `a%26b`